webapp-django/crashstats/signature/jinja2/signature/signature_report.html

```diff
@@ -13,21 +13,28 @@
 {% block content %}
 <div id="mainbody">
     <div class="page-heading">
+        <a href="#" class="display-toggle-filters toggle-filters show">Show Search Filters</a>
```

Comment:
More importantly, can we make this a button instead?
It's a nit but it's odd that it's a hyperlink. Especially since you can't right-click and "Open in a New Window" on it. 

My webpages aren't perfect but I think the only time it's "OK" to have a hyperlink instead of a button is when it can actually be a link too. Like this:
```
<a href="{page_absolute_url}" onClick={showPermalinkWidget}>Permalink</a>
```